schutzbot/deploy.sh

```diff
@@ -103,7 +103,7 @@ if [ -f "rhel8internal.repo" ]; then
     sudo mv rhel8internal.repo /etc/yum.repos.d/
     # Use osbuild from schutzfile if desired for testing custom osbuild-composer packages
     # specified by $REPO_URL in ENV and used in prepare-rhel-internal.sh
-    if [ "$SCHUTZ_OSBUILD" == 1 ]; then
+    if [ "${SCHUTZ_OSBUILD:=false}" == true ]; then
```

Comment:
What provides this variable ? Surely whatever relies on it will not expect to change the behavior behind the accepted values. 

What's the reasoning for not comparing to `1` ?

Reply:
This variable is also provided only in the scheduled pipeline settings. I've set it up so that we can use a pinned osbuild version in schutzfile if needed. This was the case when we were running the nightly testings using the osbuild-composer scratch build. osbuild was not in a new enough version in the repository so a pinned version was used instead. It should not be needed now and it again fails because of `set -u` ... There is no reason for changing it from `1` to  `true` but I had to set it up to some default value and using `true` `false` just seemed better....